model/account_dto.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+// AccountDTO represents account entity with connected data. DTO stands for Data Transfer Object.
+type AccountDTO struct {
+	*Account
+}
```

Comment:
DTOs were added to the cloud provisioner due to the complexity of adding annotations to our resources. For this reason I would keep It simple and just use `Account` for now as you probably won't need to wrap the objects for your use-case.

Reply:
Yeap that makes total sense and thanks for mentioning this. I did a cleanup of all DTOs in the code :)